en/gps_compass/README.md

```diff
@@ -1,7 +1,7 @@
-# GPS & Compass
+# Global Navigation Satellite Systems (GNSS)
```

Comment:
Note, even though this will cover both, I'm working on making it less compass-centric, and will link.